koji/collection-mash-split.py

```diff
@@ -277,9 +274,8 @@ def handle_extras(self, whole_path, mash_config, arches, extras, extras_path, gi
         for arch in arches:
             extras_cache_path = os.path.join(extras_path, mash_config, arch)
             extras_repo_path = os.path.join(whole_path, mash_config, arch, "os", "Packages")
-            extras_file = self.get_from_git(gitloc, extras_baseloc, extras + '-' + arch, extras_git_path)
-            with open(extras_file) as f:
-                self.download_extras(extras_cache_path, extras_repo_path, [line.rstrip() for line in f.readlines()])
+            extras_file = self.get_from_git(gitloc, extras_baseloc, extras + '-' + arch)
+            self.download_extras(extras_cache_path, extras_repo_path, extras_file.splitlines())
```

Comment:
This no longer strips lines in the extras file. Is that no longer needed?

Reply:
on the other hand, last time we used `extras` was in 2.1, and then we dropped it in b0522c9650645c5daaac60572426a2c7fbb78408, so maybe just nuke all that code?

Reply:
Oh, that's probably even better. We can always `git revert` if we do need it again.